frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/pages/employee/ProjectDetails/ProjectDetailsNavDetails.js

```diff
@@ -0,0 +1,49 @@
+import React, { Fragment } from "react";
+import ApplicationDetails from "../../../../../templates/ApplicationDetails";
+
+const ProjectDetailsNavDetails = ({activeLink}) => {
+
+    // integrate once api is ready
+    const workflowDetails = {}; 
+    const { applicationDetails_ProjectDetails, applicationData_ProjectDetails } = Digit.Hooks.project.useViewProjectDetails({}); 
+    const { applicationDetails_FinancialDetails, applicationData_FinancialDetails } = Digit.Hooks.project.useViewFinancialDetails({}); 
+
+    return (
+        <>
+        {
+            (activeLink === "Project_Details") && (
```

Comment:
@hariomsinha-eGov For Project Details Card pls use this component -> \micro-ui-internals\packages\modules\Estimate\src\pageComponents\ViewProject.js

We'll make this a generic component later so whenever we need to see the project details card we'll pass the projectId as prop and make a search call

Reply:
updated